pyproject.toml

```
@@ -50,6 +50,7 @@ deepseek = []
 daily = [ "daily-python~=0.18.2" ]
 deepgram = [ "deepgram-sdk~=3.8.0" ]
 elevenlabs = [ "websockets~=13.1" ]
+resembleai = [ "websockets~=13.1" ]
```

Bump version support to align with a recent change.
```suggestion
resembleai = [ "websockets>=13.1,<15.0" ]
```